03_Modules/EVDriver/src/module/api.ts

```diff
@@ -73,6 +74,10 @@ export class EVDriverModuleApi
     return this._module.authorizeRepository.createOrUpdateByQuerystring(
       request.body,
       request.query,
+    ).then((authorization) =>
+        authorization
+            ? OCPP2_0_1_Mapper.AuthenticationMapper.toAuthorizationData(authorization)
+            : undefined
```

Comment:
I would vote for returning the data model `Authorization` directly for these data endpoints.